Add files ivan

skinnyjazz opened the pull request.

skinnyjazz committed:
Add files via upload

so2niko commented on script.js:
```diff
@@ -0,0 +1,139 @@
+    return fetch(url + requestConfig).then(response => {
+        return response.json();
+    })
+}
```
```js 
const sendRequest = url => fetch(url).then(response => response.json());
```

so2niko commented on script.js:
```diff
@@ -0,0 +1,139 @@
+        }
+    }
+
+}
```
```js
const addStyleTotable = tbody => [...tbody].forEach(tr => tr.classList.toggle('green-tr', i % 2));
```

so2niko reviewed (changes requested):
1

skinnyjazz committed:
Add files via upload

-better UI
  -better table styles
  -better menu styles
-new Function
 -reversed sort
 -rangeCounter
